AppCenter/AppCenterTests/MSAlertControllerTests.m

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import "MSAlertController.h"
+#import "MSTestFrameworks.h"
+
+static NSTimeInterval const kMSTestTimeout = 1.0;
+
+@interface MSAlertControllerTests : XCTestCase
+
+@end
+
+@interface MSAlertController (Testing)
+
++ (void)makeKeyAndVisible;
+
+@end
+
+@implementation MSAlertControllerTests
+
+- (void)testMSAlertAddAction {
```

Comment:
Please fix naming and consider to add more tests

Reply:
Renamed and added some tests